diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -113,11 +116,58 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
             BLOCK -> handleLambdaBody(node)
             RETURN -> handleReturnStatement(node)
             SUPER_TYPE_LIST, VALUE_PARAMETER_LIST -> handleList(node)
+            RBRACE -> handleRightBrace(node)
```

Comment:
Shouldn't it be handled in `BlockStructureBraces` instead of `NewlinesRule`?

Reply:
Completly forgot about it. Remade bugfix